frontend/packages/ux-editor/src/components/Properties/Text.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useFormContext } from '../../containers/FormContext';
+import { useTranslation } from 'react-i18next';
+import { Heading } from '@digdir/design-system-react';
+import { EditTextResourceBindings } from '../config/editModal/EditTextResourceBindings';
+import { useLayoutSchemaQuery } from '../../hooks/queries/useLayoutSchemaQuery';
+import { useComponentSchemaQuery } from '../../hooks/queries/useComponentSchemaQuery';
+import { selectedLayoutNameSelector } from '../../selectors/formLayoutSelectors';
+import { StudioSpinner } from '@studio/components';
+import { getCurrentEditId } from '../../selectors/textResourceSelectors';
+import { TextResourceEdit } from '../TextResourceEdit';
+import { EditOptions } from '../config/editModal/EditOptions';
+import classes from './Text.module.css';
+import type { FormComponentBase } from '../../types/FormComponent';
+import type { ComponentType } from 'app-shared/types/ComponentType';
+import type { OptionsComponentBase } from 'app-shared/types/ComponentSpecificConfig';
+
+export const Text = () => {
+  const { formId, form, handleUpdate, debounceSave } = useFormContext();
+  const { t } = useTranslation();
+
+  useLayoutSchemaQuery(); // Component schema query is dependent on the data loaded by the layout schema query
+  const { data: schema } = useComponentSchemaQuery(form.type);
```

Comment:
Would it be appropriate to move useLayoutSchemaQuery into useComponentSchemaQuery to avoid having to manage this manually? 🤔